src/Life/Configuration.hs

```diff
@@ -46,6 +48,12 @@ import qualified Data.Text as T
 import qualified Text.Show as Show
 import qualified Toml
 
+-- Git branch representation.
+newtype Branch = Branch { unBranch :: Text } deriving (Show, Eq, Semigroup, Monoid)
```

Comment:
Can you explain why you need `Semigroup` and `Monoid` for `Branch`?
It seems to be a little pointless, especially `mappend` with its current behaviour doesn't make sense.

Reply:
@nixorn It's better to wrap `Branch` into `Last` monoid to add semantic of _choosing second_. `mempty` branch doesn't make sense and appending branches also weird operations. Branch names doesn't form monoid.

Reply:
Ok, thanks.